.travis.yml

```diff
@@ -53,58 +47,95 @@ notifications:
-    on:
-      branch: master
-      repo: plus3it/watchmaker
-      condition: '"$TOXENV" == *"py27"*'
+jobs:
+  include:
+    - stage: check
+      python: 3.6
+      env:
+        - TOXENV=check
+      install:
+        - pip install --upgrade tox setuptools virtualenv wheel
+      script:
+        - tox -v
+      after_failure:
+        - more .tox/log/* | cat
+        - more .tox/*/log/* | cat
+    - stage: docs
+      python: 3.6
+      env:
+        - TOXENV=docs
+      install:
+        - pip install --upgrade tox setuptools virtualenv wheel
+      script:
+        - tox -v
+      after_failure:
+        - more .tox/log/* | cat
+        - more .tox/*/log/* | cat
+    - stage: deploy
+      if: branch in (master, develop)
+      python: 2.7
+      install: pip install m2r
+      script: echo "Deploying to PyPI ..."
+      before_deploy:
+        - |
+          if [ "$TRAVIS_BRANCH" = "develop" ]; then
+            sed -i -E "s/^(version = )(.*)/\1\2.$TRAVIS_BUILD_NUMBER/" $TRAVIS_BUILD_DIR/setup.cfg
+          fi
+      deploy:
+      - provider: pypi
+        server: https://test.pypi.org/legacy/
+        distributions: sdist bdist_wheel
+        user: plus3it
+        password: $PYPI_TEST_PASSWORD
+        skip_cleanup: true
+        skip_upload_docs: true
+        on:
+          branch: develop
+      - provider: pypi
+        distributions: sdist bdist_wheel
+        user: plus3it
+        password: $PYPI_PASSWORD
+        skip_cleanup: true
+        skip_upload_docs: true
+        on:
+          tags: true
+    - stage: deploy
+      if: branch in (master, develop)
+      python: 3.6
+      install:
+        - pip install -r requirements/deploy.txt
+        - pip install --editable . # install wam, as needed by GB
+      script:
+        # create standalone application
+        - gravitybee --src-dir src --sha file --with-latest --extra-data static --verbose
+      before_deploy:
+        # prepare deployment envs
+        - source .gravitybee/gravitybee-environs.sh
+        - export SATS_TAG=$GB_ENV_APP_VERSION
+        - export SATS_BODY="* [$GB_ENV_APP_NAME v$GB_ENV_APP_VERSION CHANGELOG](https://github.com/plus3it/$GB_ENV_APP_NAME/blob/$GB_ENV_APP_VERSION/CHANGELOG.md)"
+        - export SATS_REL_NAME="Release v$GB_ENV_APP_VERSION"
+      deploy:
+        - provider: script
+          # deploy to github releases
+          script: satsuki --verbose
+          skip_cleanup: true
+          on:
+            branch: master
+        - provider: s3
+          access_key_id: $AWS_ACCESS_KEY
```

Comment:
ought to be able to remove the keys `access_key_id` and `secret_access_key`, since the SDK will read from the ENVs by default